website/content/en/status/report-2024-10-2024-12/openjdk.adoc

```diff
@@ -0,0 +1,18 @@
+=== Improve OpenJDK on FreeBSD
+
+Links: +
+link:https://freebsdfoundation.org/project/improving-openjdk-on-freebsd/[Project description] URL: link:https://freebsdfoundation.org/project/improving-openjdk-on-freebsd/[]
+
+Contact: Harald Eilertsen <haraldei-fbsd@anduin.net>
+
+The aim of this project is to improve support for Java in FreeBSD, by working with the upstream OpenJDK community, as well as the FreeBSD community in getting the changes and additions needed for fully supporting FreeBSD accepted upstream.
+
+As this is a new project, there not much to report yet, but here's what has been acheived so far:
+
+* The Java Test Regression harness (jtreg) now [builds and runs][1] on FreeBSD, and the process of link:https://github.com/openjdk/jtreg/pull/237[upstreaming the changes] has started.
```

Comment:
Is `[builds and runs][1]` meant to be a link? If so, use the `link:URL[text]` syntax as you did elsewhere.

Reply:
Ah, yes, thanks. That was an oversight. I'll fix.